koku/api/query_params.py

```diff
@@ -379,7 +368,7 @@ def get_replacement_result(param_res_list, access_list, raise_exception=True):
     """Adjust param list based on access list."""
     if ReportQueryHandler.has_wildcard(param_res_list):
         return access_list
-    if not access_list and not raise_exception:
+    if not (access_list or raise_exception):
```

Comment:
Is flipping the `and` to an `or` fixing a bug or creating one? :smiley:

Reply:
It does neither...
```python
>>> not True and not False
False
>>> not True and not True
False
>>> not False and not False
True
>>> not (True or False)
False
>>> not (True or True)
False
>>> not (False or False)
True
```
That change came from this extension: [https://sourcery.ai/](https://sourcery.ai/)